dotnet-desktop-guide/framework/wpf/advanced/how-to-remotely-survey-the-status-of-printers.md

```diff
@@ -11,6 +11,7 @@ helpviewer_keywords:
   - "remotely surveying printer status [WPF]"
   - "status [WPF], printers [WPF], surveying remotely"
 ms.assetid: d6324759-8292-4c23-9584-9c708887dc94
+description: Learn how the printer properties exposed in the APIs of Microsoft .NET Framework provide a means for performing a rapid survey of the states of printers. 
```

Comment:
"for performing a rapid survey of the states of printers."... Should states be status?